components/centraldashboard/app/api.ts

```diff
@@ -24,86 +16,31 @@ interface EnvironmentInfo {
   isClusterAdmin: boolean;
 }
 
-interface HasWorkgroupResponse {
-  user: string;
-  hasAuth: boolean;
-  hasWorkgroup: boolean;
-}
-
-
-export type SimpleRole = 'owner'| 'contributor';
-export type WorkgroupRole = 'admin' | 'edit';
-export type Role = SimpleRole | WorkgroupRole;
-export const roleMap: ReadonlyMap<Role, Role> = new Map([
-  ['admin', 'owner'],
-  ['owner', 'admin'],
-  ['edit', 'contributor'],
-  ['contributor', 'edit'],
-]);
-
 export const ERRORS = {
   operation_not_supported: 'Operation not supported'
 };
 
-export function apiError (a: {res: Response, error: string, code?: number}) {
+export function apiError(a: {res: Response, error: string, code?: number}) {
   const {res, error} = a;
   const code = a.code || 400;
   return res.status(code).json({
     error,
   });
 }
 
-/**
- * Converts Workgroup Binding from Profile Controller to SimpleBinding
- */
-export function mapWorkgroupBindingToSimpleBinding (bindings: WorkgroupBinding[]): SimpleBinding[] {
-  return bindings.map((n) => ({
-    user: n.user.name,
-    namespace: n.referredNamespace,
-    role: roleMap.get(n.roleRef.name as Role) as SimpleRole,
-  }));
-}
-
-/**
- * Converts Kubernetes Namespace types to SimpleBinding to ensure
- * compatibility between identity-aware and non-identity aware clusters
- */
-export function mapNamespacesToSimpleBinding (user: string, namespaces: V1Namespace[]): SimpleBinding[] {
-  return namespaces.map((n) => ({
-    user,
-    namespace: n.metadata.name,
-    role: roleMap.get('edit') as SimpleRole,
-  }));
-}
-
-/**
- * Converts SimpleBinding to Workgroup Binding from Profile Controller
- */
-export function mapSimpleBindingToWorkgroupBinding (binding: SimpleBinding): WorkgroupBinding {
-  const {user, namespace, role} = binding;
-  return {
-    user: {
-      kind: 'User',
-      name: user,
-    },
-    referredNamespace: namespace,
-    roleRef: {
-      kind: 'ClusterRole',
-      name: roleMap.get(role) as WorkgroupRole,
-    }
-  };
+export function NO_ROUTES(req: Request, res: Response, next: NextFunction) {
```

Comment:
Any reason for not using camel-casing for the function here? Also, given that it's only used in one place and is such a trivial function I don't see a need to have it separated and exported.

Reply:
I was leaving this exposed for future conditional routing, but yeah, for now, we can just keep it scoped to the code that exists.